src/qml/qgismobileapp.qml

```diff
@@ -670,12 +670,12 @@ ApplicationWindow {
 
     Component.onCompleted: focusstack.addFocusTaker( this )
 
-    //this is that the focus is set by selecting the empty space
+    //that the focus is set by selecting the empty space
     MouseArea {
-      enabled: parent.state!="FeatureFormEdit"
       anchors.fill: parent
       propagateComposedEvents: true
-      onClicked: {
```

Comment:
Does this mean, the `onClicked` event is not propagated to children? Are we sure, none of the child items needs the `onClicked` or `onPressed` event?